src/components/Notifications/DataAlertModal/DataAlertModal.js

```diff
@@ -68,7 +67,6 @@ export default class DataAlertModal extends React.Component {
     onManagementCreateDataAlert: () => {},
     onManagementDeleteDataAlert: () => {},
     title: 'Create Data Alert',
-    titleIcon: undefined,
```

Comment:
Do we not need these here?

Reply:
> Do we not need these here?

No, we do not need. Because you added the ` titleIcon` prop in `<Modal/>`. We can just assign some value for this prop(in `<DataAlertModal/>`)